include/CoolPropTools.h

```diff
@@ -616,50 +624,39 @@ template<class T> void normalize_vector(std::vector<T> &x)
         std::replace( file_path.begin(), file_path.end(), '\\', '/'); // replace all '\' with '/'
         
         #if defined(__ISWINDOWS__)
-        char sep = '\\';
+        const char sep = '\\'; // well, Windows (and DOS) allows forward slash "/", too :)
```

Comment:
Honestly, I don't know which WinAPIs don't accept "/". It's better to know of them, so please give an example.
But either way, I don't feel like this should be changed, as it makes no difference efficiency-wise. This looks more robust, so I have no objections, only the notice.